tools/cuds-prioritized-attribution/test/helper.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Google LLC
```

Comment:
@bipinupd

Reply:
hmm .. Sorry the file somehow gave you the impression it is only BQ related utils. It is general helper that performs the generic such as calling bash scripts to creating dataset, deleting files after a test. Since this is the only helper files I will prefer is to leave as it is.